src/pages/workspace/accounting/qbo/export/QuickbooksExportConfigurationPage.tsx

```diff
@@ -96,7 +101,7 @@ function QuickbooksExportConfigurationPage({policy}: WithPolicyProps) {
                                     shouldShowRightIcon={menuItem?.shouldShowRightIcon ?? true}
                                     onPress={menuItem?.onPress}
                                     brickRoadIndicator={menuItem?.brickRoadIndicator}
-                                    error={menuItem?.brickRoadIndicator ? translate('common.genericErrorMessage') : undefined}
+                                    errorText={menuItem?.errorText}
```

Comment:
This is still causing https://github.com/Expensify/App/pull/39191#discussion_r1570438581

Reply:
@s77rt yup - problem related to errorText component - which is not expect to be in multiline.

Reply:
Shouldn't we use `error` for now or are we going to fix the `errorText` bug here?